strings.txt

```diff
@@ -16982,6 +17012,36 @@
     pl = Użytkownicy MAPS.ME dokonali 50 000 zmian w mapach. Wspólnie tworzymy najbardziej szczegółowe mapy na świecie. Przyłącz się do nas!
     nl = MAPS.ME-gebruikers hebben 50.000 kaartwijzigingen doorgevoerd. Samen maken we de meest gedetailleerde kaarten ter wereld. Doe met ons mee!
 
+  [whatsnew_update_editor_message_update]
+    comment = Additional text in whats new
+    en = Update all maps for use editor.
```

Comment:
Игорь, одобрямс?

> On Apr 29, 2016, at 2:16 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3075#discussion_r61562845>:
> 
> > @@ -16982,6 +17012,36 @@
> >      pl = Użytkownicy MAPS.ME dokonali 50 000 zmian w mapach. Wspólnie tworzymy najbardziej szczegółowe mapy na świecie. Przyłącz się do nas!
> >      nl = MAPS.ME-gebruikers hebben 50.000 kaartwijzigingen doorgevoerd. Samen maken we de meest gedetailleerde kaarten ter wereld. Doe met ons mee!
> >  
> > +  [whatsnew_update_editor_message_update]
> > +    comment = Additional text in whats new
> > +    en = Update all maps for use editor.
> Update all maps to use Map Editor.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3075/files/1c95636bb154b14b480bf83f416967a6b8ef1a43#r61562845>